lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_builtin_method.rb

```diff
@@ -34,7 +34,7 @@ def self.miq_log_workspace(obj, _inputs)
     end
 
     def self.miq_send_email(_obj, inputs)
-      MiqAeMethodService::MiqAeServiceMethods.send_email(inputs["to"], inputs["from"], inputs["subject"], inputs["body"])
+      MiqAeMethodService::MiqAeServiceMethods.send_email(inputs["to"], inputs["from"], inputs["subject"], inputs["body"], inputs["cc"], inputs["bcc"])
```

Comment:
```
 MiqAeMethodService::MiqAeServiceMethods.send_email(inputs["to"], inputs["from"], inputs["subject"], inputs["body"], :cc => inputs["cc"], :bcc => inputs["bcc"], :content_type => inputs["content_type"])
```

Reply:
What's the point of using positional args if you're specifying which one they are here anyway?